### DISCUSSION TITLE
Follow the new layer of encryption ?

### DISCUSSION THREAD

**Discussion by ghost** [dani-garcia/vaultwarden#3169] [Q&A] [answered]
Hello i would like to ask if vaultwarden plant to follow on the announcement of the new layer of encryption from bitwarden as they mentioned on mastodon ?

I don't have checked how this "double-encrypting" field word in their thing, simply see if it will protect anything more for vaultwarden and if so if you will add them nothing more (why i ask as a Q&A and not open an idea)

source : https://fosstodon.org/@bitwarden/109745277062224768

**Comment by BlackDex** [comment, marked as answer]
That feature is only for there SaaS service. Not for there self-hosted product. Since Vaultwarden is comparable with there self-hosted product it's not something we can add as far as i can tell.

I think they use just use some azure cloud provider encryption feature to make that happen.
